README.md

```diff
@@ -69,36 +69,35 @@ to the controller-gen CLI page in the [kubebuilder documentation](https://book.k
   ```sh
   kind create cluster --image kindest/node:v1.22.4
   ```
-  &nbsp; For more advanced tests, a 3 node cluster may be more suitable.
+  &nbsp; To run the e2e tests, a 3 node cluster is necessary (due to anti-affinity).
```

Comment:
IMHO, PR should contain only changes related to the feature. I would be nice to if this doc improvement is a new PR. 
Imagine git revert <this-commit> to get Grafana back. It would result in us losing this doc improvement which isn't what we really want.

Reply:
Sure makes perfect sense :) 
https://github.com/rhobs/monitoring-stack-operator/pull/135